### PR TITLE
Module 13 Activity: Continuous Deployment Steps

### PR DESCRIPTION
**Pull request by anikeshk**
This PR contains the deployment instructions for MongoDB on MongoDB Atlas, and the server and the client on Render.com.

Preview: [/activities/continuous-deployment](https://deploy-preview-39--magical-haupia-72594e.netlify.app/activities/continuous-deployment) 

Pending:

1. Adding the GitHub Classroom invite link in the "GitHub Repository" section.

### CODE REVIEW

File: Activities/Module14 Activity.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+1. Open the [Render Dashboard](https://dashboard.render.com/).
+2. From the top menu, click on the "+ New" button and click on "Static Site".
+   1. For the Git Provider, choose your project repository. In case you do not see your project repository, re-connect your GitHub account and authorize access to your project repository.
+   2. For the Name, you can either choose an unique name OR use a name such as "cs4530-f24-XYY" (where XYY is your group number).
+   3. For the Project, select the project created earlier. For the environment, select Production or any default value.
+   4. For Branch, select "main".
+   5. For Root Directory, type in "client".
+   6. For Build Command, type in "npm install && npm run build".
+   7. For Publish directory, type in "build".
+   8. In the Environment Variables section, add a variable called `REACT_APP_API_URL`. For the value, add the server URL.
```
Comment (ViharReddy) [inline Activities/Module14 Activity.md]: change to "REACT_APP_SERVER_URL"

**Review by ViharReddy — CHANGES_REQUESTED** [review]

File: Activities/Module14 Activity.md (hunk 1 above)
Comment (anikeshk) [reply Activities/Module14 Activity.md]: I have updated the env variable name.

**Comment by anikeshk** [comment]
Added the following changes:
1. Minor grammatical mistakes.
2. Specified what default collections the student will see after they connect the deployed MongoDB cluster to their MongoDB Compass.
3. Added `fake_so` to end of the connection string.
4. Re-rephrased the environmental instructions to use the `.env` file.
5. Updated the server install and start steps.
6. Add a note about server inactivity.

**Comment by iamabhutta** [comment]
Closed without merge. Will work on #52 instead!

**Pull request closed without merging**